diff --git a/components/wrapper/CMakeLists.txt b/components/wrapper/CMakeLists.txt
index 488421a..41a88fd 100644
--- a/components/wrapper/CMakeLists.txt
+++ b/components/wrapper/CMakeLists.txt
@@ -92,7 +92,7 @@ function(add_wrapper_module)
     add_custom_command(
       OUTPUT ${STUBGEN_OUTPUT_PATHS}
       COMMAND ${STUBGEN_PATH} --package ${MODULE_NAME} --output
-              ${STUBGEN_OUTPUT_DIR}
+              ${STUBGEN_OUTPUT_DIR} --verbose
       WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
       COMMENT "Generate stubs for pybind module: ${MODULE_NAME}"
       DEPENDS ${MODULE_NAME})
